Backup Refactor Wrap-up (#1508)
spectrocloud/librarium: content/docs/04-clusters/06-cluster-management/05-backup-restore/10-add-backup-location-static.md

3. Click on the **Add New Backup Location** button. 

4. Fill out the following input fields. Refer to the table below to learn more.

	|**Field**|**Value**|
	|---|---|
	|**Location Name**|Provide a name of your choice.|
	|**Location Provider**|Select GCP from the drop-down field. |
	| **Bucket** | The name of the bucket pre-created in the GCP object store.|
	| **JSON Credentials** | Provide the JSON credentials for the external authentication of the GCP storage. Ensure the associated service account has sufficient permissions to perform the required bucket operations. |

5. Click on the **Validate** button. Palette will display a validation status message. If the validation status message indicates a success, proceed to the next step. If the validation status message indicates an error, you must review the error message and verify the configurations you have provided.  

6. Click on the **Create** button.  

You have completed configuring and adding a backup location to Palette with static credentials. 

<br /> 	

</Tabs.TabPane>

<Tabs.TabPane tab="MinIO" key="minio">

## Prerequisites

* A MinIO account.

* An S3-compliant bucket in the MinIO account. 

> * IAM policy in your MinIO account to authorize a MinIO user to perform the required read and write operations on the MinIO bucket. MinIO uses Policy-Based Access Control (PBAC) to control which IAM identities can access the resources and what actions the IAM identities are authorized to perform on the specific resources. Refer to the [MinIO Access Management] guide to learn more about the IAM policy requirements.

Author: ```suggestion
* IAM policy in your MinIO account to authorize a MinIO user to perform the required read and write operations on the MinIO bucket. MinIO uses Policy-Based Access Control (PBAC) to control which IAM identities can access the resources and what actions the IAM identities are authorized to perform on the specific resources. Refer to the [MinIO Access Management](https://min.io/docs/minio/linux/administration/identity-access-management/policy-based-access-control.html#access-management) guide to learn more about the IAM policy requirements.
```

3. Click on the **Add New Backup Location** button.

4. Fill out the following input fields. Refer to the table below to learn more.

	|**Field**|**Value**|
	|---|---|
	|**Location Name**|Provide a name of your choice.|
	|**Location Provider**|Select MinIO from the drop-down field. |
	|**Certificate**|Service provider certificate, if your organization prefers it.|
	|**S3 Bucket**|S3 bucket name must be pre-created in the MinIO object-store. |
	| **Region** | Region in which MinIO server is configured. Example: `us-east-1` |
	|**S3 URL** | URL of the MinIO object storage console. Example: `http://12.123.234.567:0000`|
	|**Force S3 path style** | To force S3 path style addressing or else the URL will be converted to virtual-hosted style addressing with bucket name appended to the URL. This is an optional setting. |

5. Next, provide the access key for the MiniIO user. The access key will have two parts - the access key ID and the secret key.  

6. Click on the **Create** button. 

You have completed configuring and adding a backup location to Palette with static credentials.   

<br />	

</Tabs.TabPane>

<Tabs.TabPane tab="Azure" key="azure">

## Prerequisites

> * An active Azure cloud account. You will need to be aware of the values for the following items:

Author: ```suggestion
* An active Azure cloud account. You will need the following Azure items to complete the backup setup:
```
